proto/regen/ecocredit/v1/events.proto

```diff
@@ -130,3 +130,40 @@ message EventCancel {
   // amount is the decimal number of credits that have been cancelled.
   string amount = 3;
 }
+
+// EventClassAdminUpdated is emitted when the admin address of a credit class is changed.
+message EventClassAdminUpdated {
+
+  // class_name is the name of the class that was updated.
+  string class_name = 1;
+
+  // old_admin is the admin of the credit class before the update.
+  string old_admin = 2;
+
+  // new_admin is the admin of the credit class after the update.
+  string new_admin = 3;
+}
+
+// EventClassIssuersUpdated is emitted when the issuer list for a credit class is updated.
+message EventClassIssuersUpdated {
+  // class_name is the name of the class that was updated.
+  string class_name = 1;
+
+  // added_issuers contains all the addresses added to the class issuer list.
+  repeated string added_issuers = 2;
+
+  // removed_issuers contains all the addresses removed from the class issuer list.
+  repeated string removed_issuers = 3;
+}
+
+// EventClassMetadataUpdated is emitted when the credit class metadata is changed.
+message EventClassMetadataUpdated {
+  // class_name is the name of the class that was updated.
+  string class_name = 1;
+
+  // old_metadata is the metadata before the update.
+  string old_metadata = 2;
```

Comment:
not sure if its worth to have this field, lmk your opinions!

Reply:
I don't have a strong opinion here but including it seems reasonable.